Microsoft/microsoft-intune/ingest/parser.yml

```
@@ -0,0 +1,57 @@
+name: microsoft-intune
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+  - name: parsed_date
+    external:
+      name: date.parse
+      properties:
+        input_field: '{{json_event.message.time}}'
+        output_field: datetime
+  - name: set_common_fields
+  - name: action_properties
+    filter: "{{json_event.message.properties != null}}"
+
+stages:
+  set_common_fields:
+    actions:
+      - set:
+          '@timestamp': '{{parsed_date.datetime}}'
+          action.description: '{{json_event.message.properties.Description}}'
+          action.name: '{{json_event.message.operationName}}'
+          action.target: 'user'
+          action.type: '{{json_event.message.category}}'
+          event.kind: 'event'
+          event.type: ['info']
+          host.id: '{{json_event.message.properties.DeviceId}}'
+          host.ip: '{{json_event.message.properties.WifiMacAddress}}'
+          host.mac: '{{json_event.message.properties.WifiMacAddress}}'
+          host.name: '{{json_event.message.properties.ManagedDeviceName or json_event.message.properties.DeviceHostName }}'
+          host.type: '{{json_event.message.properties.Model}}'
+          intune.compliant: '{{json_event.message.properties.CompliantState}}'
+          network.application: '{{json_event.message.ApplicationName}}'
+          os.full: '{{json_event.message.properties.OS}}'
+          os.version: '{{json_event.message.properties.OSVersion}}'
+          service.name: '{{json_event.message.properties.ManagedBy}}'
+          source.ip: '{{json_event.message.actor.ipAddress}}'
+          source.mac: '{{json_event.message.properties.WifiMacAddress}}'
+          user.email: '{{json_event.message.properties.UserEmail}}'
+          user.id: '{{json_event.message.properties.IntuneAccountId}}'
+          user.name: '{{json_event.message.properties.UserName or json_event.message.properties.Actor.UPN}}'
```

Maybe UPN should land in `user.email`?
```suggestion
          user.name: '{{json_event.message.properties.UserName}}'
          user.email: '{{ json_event.message.properties.Actor.UPN}}'
```